buzzmobile/simulation/sim_car_interface/src/sim_car_interface.cpp

```diff
@@ -35,13 +35,13 @@ double speed_measured_right = 0.0;
 double steer_set_point = 0.0;
 
 // TODO(sahit): change to rosparams
-constexpr double chassis_length = 0.33246;
-constexpr double chassis_width = 0.28732;
+constexpr double chassis_length = 1.8; 
+constexpr double chassis_width = 1.57;
```

Comment:
@joshuamorton yeah the dims don't match at all and these were the numbers I got from RJ...
In the CAD those are the dimensions of the car, when (if) I get access to the car, I'll check myself which measurements are correct. For now we should merge it in but with a TODO.